safe/impact_function/provenance_utilities.py

```diff
@@ -28,17 +28,17 @@ def get_map_title(hazard, exposure, hazard_category):
     :rtype: str
     """
     if hazard == hazard_generic:
-        map_title = tr('{exposure_name} affected'.format(
+        map_title = tr(u'{exposure_name} affected'.format(
```

Comment:
Ok, I will work on that in another PR. Let's keep this one for reporting standards